pkg/xds/conv/convertxds.go

```diff
@@ -599,6 +600,9 @@ func convertFilterChainsAndGetRawFilter(xdsListener *xdsapi.Listener) ([]v2.Filt
 		if xdsFilterChain.GetFilterChainMatch() != nil {
 			chainMatch = xdsFilterChain.GetFilterChainMatch().String()
 		}
+
+		// collect tls configs
+		tlsConfigs = append(tlsConfigs, convertTLS(xdsFilterChain.GetTlsContext()))
```

Comment:
@wZH-CN  xdsFilterChain.GetTlsContext is deprecated and do not use, Now we should use `xdsFilterChain.GetTransportSocket().GetTypedConfig()` instead of `xdsFilterChain.GetTlsContext()`， forexample:
``` go	
    downstreamTLSContext := &auth.DownstreamTlsContext{}
    err := ptypes.UnmarshalAny(xdsFilterChain.GetTransportSocket().GetTypedConfig(), downstreamTLSContext)
    if err != nil {
        xxx
    }
    convertTLS(downstreamTLSContext)
```

Reply:
FYI https://github.com/envoyproxy/go-control-plane/blob/v0.9.4/envoy/api/v2/listener/listener_components.pb.go#L265